frontends/docs/[id]/DocumentationPage.tsx

```diff
@@ -90,7 +90,7 @@ export const DocumentationPage: FC = () => {
           </div>
         )}
         res={fetchDocRes}
-        className='editor-container'
+        className='editor-container px-2'
```

Comment:
padding should be added only for mobile screens. use px-xs-2 class (create one if needed)